readthedocs/rtd_tests/tests/test_backend.py

```diff
@@ -254,6 +265,17 @@ def test_update_and_checkout(self):
         self.assertEqual(code, 0)
         self.assertTrue(exists(repo.working_dir))
 
+    def test_git_checkout_invalid_revision(self):
```

Comment:
The test is duplicated.

I think you moved it and forget to remove the old one.

Reply:
It's not duplicated, this one is for mercurial, forgot to change the name p: